Documentation/design-docs/DotNetCore-SharedPackageStore.md

```diff
@@ -117,14 +106,16 @@ The host will probe in the following order of precedence for `dotnet run` and ap
 
 + `$CORE_SERVICING` on Unix or `%ProgramFiles(x86)%\coreservicing` on Windows.
```

Comment:
This ordering, in general, does not match the way things work. We should fix it. The host probes the locations in the following order:

1) Servicing
2) AppBin (including SharedFX)
3) Store locations
4) Additional Probing Paths

I think we should list the actual order so that readability and understanding of the concept remains simple. For specifics of each bullet (e.g. EXE or Global, one probe location over the other), we should have sub-bullets to explain those.